exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/SqlConverter.java

```diff
@@ -139,6 +151,24 @@ public SqlNode parse(String sql) {
       SqlParser parser = SqlParser.create(sql, parserConfig);
       return parser.parseStmt();
     } catch (SqlParseException e) {
+
+      // Attempt to use default back_tick quote character for identifiers when
```

Comment:
One of the requirements was still working identifiers with backticks even while ANSI_QUOTES mode is on.
`sql.contains("`")` condition isn't mandatory here. It was removed in a new commit.